### PR TITLE
[BLZT-4] Added Configuration and Configuration Beans

### PR DESCRIPTION
**Pull request by mvorotniak**

### CODE REVIEW

File: core/src/main/java/com/bobocode/bring/core/anotation/resolver/impl/ConfigurationBeanNameAnnotationResolver.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.bobocode.bring.core.anotation.resolver.impl;
+
+import com.bobocode.bring.core.anotation.Component;
+import com.bobocode.bring.core.anotation.Configuration;
+import com.bobocode.bring.core.anotation.resolver.AnnotationResolver;
+
```
Comment (andrewpikozh) [inline core/src/main/java/com/bobocode/bring/core/anotation/resolver/impl/ConfigurationBeanNameAnnotationResolver.java:6]: Need to provide some information in javadoc
Maybe something like this. What do you think ?
Screenshot 2023-11-13 at 10 50 37

**Review by andrewpikozh — CHANGES_REQUESTED** [review]

File: core/src/main/java/com/bobocode/bring/core/anotation/resolver/impl/ConfigurationBeanNameAnnotationResolver.java (hunk 1 above)
Comment (marianaavorotniak) [reply core/src/main/java/com/bobocode/bring/core/anotation/resolver/impl/ConfigurationBeanNameAnnotationResolver.java:6]: I want to add Javadocs everywhere a bit later, when everything is more stable, so Javadoc is not rewrited several times

Comment (mvorotniak) [reply core/src/main/java/com/bobocode/bring/core/anotation/resolver/impl/ConfigurationBeanNameAnnotationResolver.java:6]: I want to add Javadocs everywhere a little bit later after more detailed testing, when everything is more stable, so Javadoc is not rewrited several times.
I will add some documentation in next PR with tests

Comment (andrewpikozh) [reply core/src/main/java/com/bobocode/bring/core/anotation/resolver/impl/ConfigurationBeanNameAnnotationResolver.java:6]: Okay, no problem )

**Review by andrewpikozh — APPROVED** [review]

**Pull request merged**